util/reggen/field.py

```diff
@@ -381,3 +382,18 @@ def apply_alias(self, alias_field: 'Field', where: str) -> None:
         # This gives us a way to check whether a register has been overridden
         # or not, and what the name of the original register was.
         self.alias_target = alias_field.alias_target
+
+    def scrub_alias(self, where: str) -> None:
+        '''Replaces sensitive fields in field with generic names
+
+        This function can be used to create the generic field descriptions
+        from full alias hjson definitions. It will only work on fields
+        where the alias_target keys are defined, and otherwise throw an error.
```

Comment:
I'm a bit confused: I can't see any way in which the code below can throw an error?

Reply:
I've reused the same description as for other `scrub_alias` functions - but in the `field` class this sentence does indeed not make any sense - hence this has been removed from the description.